Ical.Net/Serialization/DataTypes/WeekDaySerializer.cs

```diff
@@ -30,22 +31,33 @@ public override string SerializeToString(object obj)
         {
             value += ds.Offset;
         }
-        value += Enum.GetName(typeof(DayOfWeek), ds.DayOfWeek).ToUpper().Substring(0, 2);
+
+        try
+        {
+            var name = Enum.GetName(typeof(DayOfWeek), ds.DayOfWeek);
+            value += name!.ToUpper().Substring(0, 2);
```

Comment:
If `name` is null, it would throw and be catched silently, which is ok, but not too readable. Handle explicitly?

Reply:
Agree, `if (name == null) return null; is more readable